lib/client.js

```diff
@@ -42,30 +43,40 @@ class Client {
   }
 
   async deleteJobs(company, options = {}) {
+    const versions = [1, 4];
     const {
-      url = "https://api.peviitor.ro/v4/clean/",
+      url = "https://api.peviitor.ro",
       contentType = "application/x-www-form-urlencoded",
     } = options;
     const dataObject = { company };
     const header = "Content-Type";
     this.#setHeader(header, contentType);
-    await this.post(url, dataObject)
-      .then(console.log(`Cleaned jobs for ${company}`))
+    await Promise.all(
```

Comment:
is this intended?
Promise.all fails fast, and we wont be able to tell immediately if v1 or v4 is the culprit.

if you want better logging, use promise.allsettled. you won't have a ".catch" method since promise.allsettled always resolves, but you can check response.status.
if you don't care (say, v1 and v4 behave exactly the same, belong to same app, etc) then leave it as promise.all

Reply:
Response status is always the same, you will not see any error from the api